build.cmd

```diff
@@ -33,6 +33,8 @@ IF %BUILDCMD_DNX_VERSION%=="" (
 	CALL packages\KoreBuild\build\dnvm install %BUILDCMD_DNX_VERSION% -runtime CLR -arch x86 -alias default
 )
 CALL packages\KoreBuild\build\dnvm install default -runtime CoreCLR -arch x86
+CALL packages\KoreBuild\build\dnvm install default -runtime CLR -arch x64
```

Comment:
Well, these lines are mandatory for testing this repo and 19f8830 was a blanket change that overwrote most `build.cmd` files.